### PR TITLE
Update travis, Readme and remove .coveragerc

### PR DESCRIPTION
**Pull request by bwrsandman**

### CODE REVIEW

**Comment by yvaucher** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+[![Build Status](https://travis-ci.org/OCA/account-payment.svg?branch=7.0)](https://travis-ci.org/OCA/account-payment)
+[![Coverage Status](https://coveralls.io/repos/OCA/account-payment/badge.png?branch=7.0)](https://coveralls.io/r/OCA/account-payment?branch=7.0)
+
 OpenERP account payment modules
```
Comment (pedrobaeza) [inline README.md:4]: You can also get the occassion to expand README text including OCA word in the title (something like `OCA account payment modules for OpenERP`), and put the disclaimer in the footer:

```
----

OCA, or the Odoo Community Association, is a nonprofit organization whose 
mission is to support the collaborative development of Odoo features and 
promote its widespread use.

http://odoo-community.org/
```

Comment (bwrsandman) [inline README.md:4]: Without a description, there isn't much I can do about this.

I can add the disclaimer, but could someone who knows this repo well supply the description?

Comment (yvaucher) [inline README.md:4]: http://odoo-community.org/project/10 ? More a summary than a description. Better than nothing?

**Comment by tafaRU** [comment]
:+1:

**Comment by pedrobaeza** [comment]
You can put this text:

```
This repository hosts modules for handling requirements related to payment topics:
* Extend payment orders for direct debit also.
* Add multiple payment days.
* Include payment types at partner, sales and purchases levels.
* Etc.
```

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**